### PR TITLE
add new category and tesults

### PR DESCRIPTION
**Pull request by ajeetd**
Adds a new category called 'Test Results Reporting and Test Case Management' and also adds tesults.com with details about free plans for open source software devs and other developers.

### CODE REVIEW

**Comment by ripienaar** [comment]
reluctant to add a 1 entry category tbh, but also cant see anywhere else its a natural fit - any suggestion other than a new category?

**Comment by ajeetd** [comment]
I agree that this doesn't easily fit into the existing categories, I looked at them all before creating the new one. I also do not know of any SaaS in the same space that has a free tier so I cannot suggest any additions. I could just add it to Monitoring or Miscellaneous but it wouldn't be easy to find if someone looking for testing related tools. I think some broader or general Testing related category needs to exist outside of Automated Browser Testing, considering testing is such big area in software development. Perhaps it doesn't need to be as specific as test results reporting and test case management.

**Comment by ripienaar** [comment]
I also thought we could make a testing area or expand ci to include these. 

Do you feel up for adjusting to make a general testing section?

**Comment by ajeetd** [comment]
Sure, I can make a section called 'Testing'. Would you like other test tools to be moved to this? Such as the Automated Browser Testing ones?

**Comment by ripienaar** [comment]
yup, move all those related into it :)

File: README.md [hunk 1]
```diff
@@ -49,6 +49,7 @@ Table of Contents
    * [Source Code Repos](#source-code-repos)
    * [Storage and Media Processing](#storage-and-media-processing)
    * [STUN, WebRTC, Web Socket Servers and Other Routers](#stun-webrtc-web-socket-servers-and-other-routers)
+   * [Test Results Reporting and Test Case Management](#test-results-reporting-and-test-case-management)
```
Comment (ripienaar) [inline README.md]: this line can now go away right?

Comment (ajeetd) [reply README.md]: Sorry about that, done.

**Comment by ripienaar** [comment]
thank you for sticking with it, really appreciate it :)

**Pull request merged**